flang/CMakeLists.txt

```diff
@@ -228,6 +228,11 @@ if (FLANG_STANDALONE_BUILD)
     add_custom_target(doxygen ALL)
   endif()
 
+  # Override the value from installed CMake files, as they refer
+  # to the directory used during the original MLIR package build,
+  # which may be no longer available.  Instead, use the current checkout.
+  set(MLIR_MAIN_SRC_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../mlir )
```

Comment:
This change is redundant, it has already been fixed by https://github.com/llvm/llvm-project/pull/125842.

Reply:
Oh, thanks for letting me know. I'm going to test if I can revert it cleanly in a minute.